### PR TITLE
Issue #13 - Coverage check

### LINKED ISSUE #13: Python script: coverage tests for projects
### What is the expected behavior?

Implement function to run coverage tests for specified repository against stable version of qiskit.

See [python runner](https://github.com/qiskit-community/ecosystem/pull/8/files#diff-2a724697aa06f1db8af7b6184d0714048fb567e09688288ef3d885c9e19e9bc1R71) for getting the idea. 

Runner should handle execution workflow for coverage and return summary

**Related stories:**  
- as core team member/curator I want CI processes takes care of tests, style checks, coverage after submission so that it would reduce time of review of submission
- as core team member/curator I want checks and tests to have Terra version associated with them so that we can track which version is breaking the code
- as core team member/curator I want CI for tests/checks to be executed in timely manner (weekly) so that entire ecosystem always remains up-to-date

**Epic** #9

### PR DESCRIPTION
**Pull request by mickahell**
### Summary
Add test coverage for candidate submission & edit stuff for styles check

- [x] Codebase
- [x] Template tox
- [x] Template .coveragerc 
- [x] Tests
- [x] Add forgetting stuff about styles check
- [x] Up the duplicate code option from 10 -> 17

### Details and comments
- Add styles default command and templating autom
- Add more templating tests
- Had to up the duplicate code option from 10 to 17, because the 2 runners class are very similar

Closes #13 
Epic #9

### CODE REVIEW

**Comment by mickahell** [comment]
@IceKhan13 I understand, black is the code formatter but how can I correct the files reading ?

**Comment by mickahell** [comment]
@IceKhan13 now it's better ;) can you review :D ?

File: tests/runners/test_runner.py [hunk 1]
```diff
@@ -60,3 +60,14 @@ def test_styles_runner_on_configured_repo(self):
...
+    def test_coverages_runner_on_configured_repo(self):
+        """Configured repo runner test."""
+        runner = PythonCoverageRunner(
+            "test", working_directory=self.configured_project_dir
+        )
+
+        runner.cloned_repo_directory = self.configured_project_dir
+        _, result = runner.workload()
+
+        self.assertTrue(len(result) > 0)
```
Comment (IceKhan13) [inline tests/runners/test_runner.py:73]: Later on we will need to fix this tests by refactoring resource repositories. We need to have one repository which does not comply with tests, one which will comply, one which comply with config and one without.

**Review by IceKhan13 — APPROVED** [review]
Looks legit!

**Pull request merged**